source/connections-survive-step-down/tests/README.rst

```diff
@@ -0,0 +1,209 @@
+===========================================
+Connections Survive Primary Step Down Tests
+===========================================
+
+These tests can be used to verify a driver's compliance with server discovery
+and monitoring requirements with respect to handling "not master" and
+"server shutting down" error responses from the server.
+
+These tests apply only to replica set topologies.
+
+Tests
+-----
+
+The driver should implement the following tests:
+
+getMore Iteration
+`````````````````
+
+This test requires server version 4.2 or higher, and makes use of events
+defined in the `CMAP specification
+<https://github.com/mongodb/specifications/blob/master/source/connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst>`_.
+
+Perform the following operations:
+
+- Insert 5 documents into a collection.
+- Clear the connection pool to the current primary (see CMAP spec for
+  the definition of connection pool clear operation, and
+  `Why clear the pool in getMore test?`_ for rationale).
+- Start a find operation on the collection with a batch size of 2, and
+  retrieve the first batch of results.
+- Verify that a ConnectionCreated CMAP event has been published.
+- Send a `replSetStepDown` command to the current primary and verify that
+  the command succeeded.
+- Retrieve the next batch of results from the cursor obtained in the find
+  operation, and verify that this operation succeeded.
+- Verify that no new ConnectionCreated CMAP events have been published.
+
+Not Master - Keep Connection Pool
+`````````````````````````````````
+
+This test requires server version 4.2 or higher.
+
+Perform the following operations:
+
+- Perform server selection for the primary server to obtain a server object
+  with an initialized connection pool.
+- Mock an operation returning NotMaster operation failure error code.
+- Verify that:
+  - Connection pool generation (per CMAP spec) has not changed
```

Comment:
If you view GitHub's rendering of the RST file, this does not render as a nested sub-list. Your alignment is correct, but per [Bullet Lists](http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#bullet-lists) I think sub-lists require a line break above and below.

Reply:
I see, thank you.